translation-bounties-2021.md

```diff
@@ -0,0 +1,33 @@
+# Translation Bounties Program (2020) — Archived
+
+**We want Sora to speak the languages of the world and unite it.**
+
+For that, we started our translation program – you can translate, review translations and receive VAL tokens as a reward for your help.
+We use [Lokalise](https://lokalise.com/) for translations and it makes it really easy to translate.
+Translation includes Fearless Wallet, Sora project translation and Capital project translation (it is a module for Sora iOS application).
+
+[Translation Round Q1 2021 Article](https://sora-xor.medium.com/rewards-for-sora-ecosystem-translations-c69bc38e931a)
+
+### Here is how it works:
+1. Join our [Translation Group on Telegram](https://t.me/sora_translation_community)
+2. Check out the table below – if you want to translate or review a language that is not yet translated/ reviewed – let us know about it in the group.
+We will check and ask you to send us your email and name to add you to Lokalise. Don't hesitate to let us know if you can't access some features you need.
```

Comment:
Should we mention form to fill in? like in the article?

Reply:
@zilenka I thought about it but afaik, currently Pietro does not ask to fill in the form...

Reply:
I sent him the link to this PR to take a look

Reply:
I always ask people to fill the form because it helps me keeping track of their work and task, but I always prefer they reach me out first. So I think we can leave it as it is

Reply:
I think we should also add this link: https://medium.com/sora-xor/rewards-for-sora-ecosystem-translations-c69bc38e931a.
For example we can say that at this link you can see updated rewards.
This is the existing sheet with completed languages, it is very minimal and not fancy but it works https://docs.google.com/spreadsheets/d/1BBWsXk27DdAnvJVlsHOkW88SX8pnwqq1kI-bkHEV73E/edit#gid=0
The rest i okay to me :D

Reply:
The link in on line 9 but sure, I can say about the rewards there too. About the form — maybe if you prefer to be contacted first, we could give them your Telegram alias?

Reply:
Also, this GitHub page is made to replace the sheet of existing languages, so I do not think we need that link.